js/front-panel.js

```diff
@@ -230,7 +230,8 @@ class FrontPanel {
           const [fromIdx, toIdx] = [connection.from, connection.to].sort((a, b) => b - a)
           const from = this.connectors[fromIdx];
           const to = this.connectors[toIdx];
-          return from.name + " -> " + to.name;
+          const color = connection.color;
+          return "<span style='color: " + color + "'>" + from.name + " -> " + to.name + "</span>";
```

Comment:
Hi @zykure , thank you for your PR's, much appreciated! Starting with this one:

I like the idea, but the colored text does not always match with the background, for some colors it's hard to read. Just too many colors. But I like the idea and have a counter-proposal to only make the in-between arrow colored. I think it gives it just that nice touch. What do you think?

```suggestion
return `${from.name}<span style="color: ${color}"> -> </span>${to.name}`;
```